includes/ui/tabs/overview.php

```diff
@@ -7,6 +7,8 @@
 
 defined( '\\ABSPATH' ) || exit;
 
+/** @var \Rhubarb\RedisCache\Plugin $roc */
+
```

Comment:
Let's drop this space.

Reply:
This "defines" `$roc`

I leave it 100% up to you.

Reply:
When it touches the next line: there will be Confusion.